### ISSUE TITLE
Some fields in config file are ignored

### ISSUE THREAD

**Issue by arndey** [hyperledger/iroha#1635]
```Hello! I can't configure genesis through
"GENESIS": {
    "ACCOUNT_PUBLIC_KEY": ...,
    "ACCOUNT_PRIVATE_KEY": {...}
} 
```
field like in [doc](https://github.com/hyperledger/iroha/blob/iroha2-dev/docs/source/references/config.md).
Got errors 
```"0: Failed to initialize genesis. 1: Genesis account public key is empty."
But I can do it through "GENESIS_CONFIGURATION": {
    "GENESIS_ACCOUNT_PUBLIC_KEY": ...,
    "GENESIS_ACCOUNT_PRIVATE_KEY": {...}
}
```

According to the logs I can set trusted peers through env var `SUMERAGI_TRUSTED_PEERS`, but can't through config file field `SUMERAGI.TRUSTED_PEERS`

Same problems with `MAX_LOG_LEVEL`, `TORII_P2P_ADDR`, `TORII_API_URL`

**Comment by appetrosyan** [comment]
MWE:

docker-compose.yml
```
services:
      iroha:
    image: hyperledger/iroha2:dev-nightly-9449b16f8f89cfa8323dfc6c101d2a07288c076d
#    image: iroha1/iroha:iroha2-dev
    environment:
      TORII_P2P_ADDR: iroha:1337
      TORII_API_URL: iroha:8080
      MAX_LOG_LEVEL: INFO
#      IROHA_PUBLIC_KEY: "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
#      IROHA_PRIVATE_KEY: '{"digest_function": "ed25519", "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"}'
#      SUMERAGI_MAX_FAULTY_PEERS: 0
#      SUMERAGI_TRUSTED_PEERS: '[{"address":"iroha:1337", "public_key": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"}]'
      RUST_BACKTRACE: full
    volumes:
      - "./genesis.json:/genesis.json"
      - "./config.json:/config.json"
    command: ./iroha_cli --genesis-path genesis.json --submit-genesis
```

Config.json

```
{
  "PUBLIC_KEY": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
  "PRIVATE_KEY": {
    "digest_function": "ed25519",
    "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
  },
  "KURA": {
    "INIT_MODE": "strict",
    "BLOCK_STORE_PATH": "./blocks",
    "BLOCKS_PER_STORAGE_FILE": 1000,
    "MAILBOX": 100
  },
  "SUMERAGI": {
    "PEER_ID": {
      "address": "iroha:1337",
      "public_key": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
    },
    "TRUSTED_PEERS": [
      {
        "address": "iroha:1337",
        "public_key": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
      }
    ],
    "BLOCK_TIME_MS": 1000,
    "COMMIT_TIME_MS": 1000,
    "TX_RECEIPT_TIME_MS": 200,
    "N_TOPOLOGY_SHIFTS_BEFORE_RESHUFFLE": 1,
    "MAX_INSTRUCTION_NUMBER": 4096,
    "MAILBOX": 100
  },
  "TORII": {
    "P2P_ADDR": "iroha:1337",
    "API_URL": "iroha:8080",
    "MAX_TRANSACTION_SIZE": 32768,
    "MAX_SUMERAGI_MESSAGE_SIZE": 16384000,
    "MAX_INSTRUCTION_NUMBER": 4096
  },
  "BLOCK_SYNC": {
    "GOSSIP_PERIOD_MS": 10000,
    "BATCH_SIZE": 4,
    "MAILBOX": 100
  },
  "QUEUE": {
    "MAXIMUM_TRANSACTIONS_IN_BLOCK": 8192,
    "MAXIMUM_TRANSACTIONS_IN_QUEUE": 65536,
    "TRANSACTION_TIME_TO_LIVE_MS": 86400000,
    "FUTURE_THRESHOLD_MS": 1000
  },
  "LOGGER": {
    "MAX_LOG_LEVEL": "ERROR",
    "TELEMETRY_CAPACITY": 1000,
    "COMPACT_MODE": false,
    "LOG_FILE_PATH": null
  },
  "GENESIS_CONFIGURATION": {
    "GENESIS_ACCOUNT_PUBLIC_KEY": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
    "GENESIS_ACCOUNT_PRIVATE_KEY": {
      "digest_function": "ed25519",
      "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
    }
  },
  "WSV": {
    "ASSET_METADATA_LIMITS": {
      "max_len": 1048576,
      "max_entry_byte_size": 4096
    },
    "ASSET_DEFINITION_METADATA_LIMITS": {
      "max_len": 1048576,
      "max_entry_byte_size": 4096
    },
    "ACCOUNT_METADATA_LIMITS": {
      "max_len": 1048576,
      "max_entry_byte_size": 4096
    },
    "DOMAIN_METADATA_LIMITS": {
      "max_len": 1048576,
      "max_entry_byte_size": 4096
    },
    "IDENT_LENGTH_LIMITS": {
      "min": 1,
      "max": 128
    }
  },
  "TELEMETRY": {
    "NAME": null,
    "URL": null,
    "MIN_PERIOD": 1,
    "MAX_EXPONENT": 4,
    "FILE": null
  },
  "NETWORK": {
    "MAILBOX": 100
  }
}
```

**Comment by appetrosyan** [comment]
Fixed in 2.0.0.pre.rc.1
